vaadin-testbench-core/src/test/java/com/example/base/About.java

```diff
@@ -17,4 +17,15 @@
 @Tag("div")
 @Route("about")
 public class About extends Component {
+    long id = System.currentTimeMillis();
+
+    @Override
+    public boolean equals(Object obj) {
+        return obj instanceof About && this.id == ((About) obj).id;
+    }
+
+    @Override
+    public int hashCode() {
+        return super.hashCode();
+    }
```

Comment:
Can be removed, since it only invokes super? Or is the method overridden for some specific purpose?